homework/week1/RosmaryFC/HoroscopeHW20150508/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,152 @@
+<resources>
+    <string name="app_name">HoroscopeHW20150508</string>
+
+    <string name="title_section1">Home</string>
+    <string name="title_section2">Signs</string>
+    <string name="title_section3">Type your Birthday</string>
+    <string name="title_section4">Game</string>
+
+    <string-array name="signs_array">
```

Comment:
This is great when you made the array. One suggestion was make a hashmap with the key as the zodiac name and the value as the element sign....but if this make you understand it better, it's still good :D